frontend/components/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "vue-strap",
+  "version": "1.1.36",
+  "description": "Bootstrap components built with Vue.js",
+  "main": "dist/vue-strap.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MarkBind/markbind.git"
+  },
+  "homepage": "https://markbind.org",
+  "dependencies": {
+    "bootstrap": "^3.3.7",
+    "jquery": "^3.3.1",
+    "terser-webpack-plugin": "^2.3.5",
+    "vue": "^2.0.0",
+    "vue-loader": "^15.9.1",
+    "webpack": "^4.42.1",
+    "webpack-merge": "^4.2.2"
+  },
+  "files": [
+    "LICENSE",
+    "README.md",
+    "CHANGELOG.md",
+    "src",
+    "dist"
+  ],
+  "keywords": [
+    "bootstrap",
+    "vue-bootstrap",
+    "vue-component",
+    "vue",
+    "markbind"
+  ],
+  "devDependencies": {
+    "@babel/core": "^7.9.0",
+    "@babel/plugin-transform-runtime": "^7.9.0",
+    "@babel/preset-env": "^7.9.0",
+    "@babel/runtime": "^7.9.2",
+    "@vue/test-utils": "^1.0.0-beta.33",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^25.4.0",
+    "babel-loader": "^8.1.0",
+    "css-loader": "^3.4.2",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-vue": "^6.2.2",
+    "expose-loader": "^0.7.5",
+    "jest": "^25.4.0",
+    "jest-serializer-vue": "^2.0.2",
+    "json-loader": "^0.5.7",
+    "less": "^3.11.1",
+    "less-loader": "^5.0.0",
+    "node-sass": "^4.13.1",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.1.3",
+    "vue-hot-reload-api": "^2.3.4",
+    "vue-jest": "^3.0.5",
+    "vue-template-compiler": "^2.6.11",
+    "webpack-cli": "^3.3.11",
+    "webpack-dev-server": "^3.10.3"
+  },
+  "scripts": {
+    "build": "webpack --progress --hide-modules --config  webpack.build.js",
+    "test": "jest",
+    "updatesnapshot": "jest --updateSnapshot",
+    "builddev": "webpack --progress --hide-modules --config  webpack.dev.js",
+    "lint": "./node_modules/.bin/eslint . --ext .js,.vue; exit 0",
+    "lintwin": "./node_modules/.bin/eslint . --ext .js,.vue",
+    "lintfix": "./node_modules/.bin/eslint . --fix --ext .js,.vue"
+  },
```

Comment:
Do we want to add helper commands (e.g. `vuebuild`, `vuelint`, `lintall`) that `cd frontend/components` and then run these in the `"scripts"` of the root package.json?
Or do we plan to do it in a separate PR?

Reply:
if we go with removing `package.json` and use nested eslintrcs that won't be necessary, but either way a separate PR would be better.
Right now i'm aiming for "copy and paste + make ci work + edit whatever documentation dosen't make sense anymore" (I'll add in devdocs as well once #1246 gets merged)